examples/scripts/deconv_datagen_foam1.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+Generation of Blurred Data for NN Training
```

Comment:
For symmetry (and clarity), it would be better to also have something here corresponding to the `(Natural Images)` in the corresponding example.